sdk/webpubsub/azure-messaging-webpubsubservice/examples/send_messages_aad.py

```diff
@@ -0,0 +1,51 @@
+import io
+import logging
+import os
+
+from azure.messaging.webpubsubservice import WebPubSubServiceClient
+from azure.identity import ClientSecretCredential
+
+logging.basicConfig(level=logging.DEBUG)
+LOG = logging.getLogger()
+
+# Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+# AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET, WEBPUBSUB_ENDPOINT
+try:
+    tenant_id = os.environ.get("AZURE_TENANT_ID")
```

Comment:
You shouldn't need to get all of these environment variables, using `DefaultAzureCredential` will get them for you.

See [here](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/samples/sample_authentication.py#L58) for an example